src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2253,21 +2231,87 @@ public function should_queue_import( $should_queue_import = null ) {
 	 *
 	 * @since TBD
 	 *
-	 * @param int $organizer_id The organizer post ID.
-	 * @param string $service_image
+	 * @param int                                         $organizer_id The organizer post ID.
+	 * @param string                                      $image_url
 	 * @param Tribe__Events__Aggregator__Record__Activity $activity
 	 *
 	 * @return bool Whether the image was attached to the organizer or not.
 	 */
-	public function import_organizer_image( $organizer_id, $service_image, $activity ) {
+	public function import_organizer_image( $organizer_id, $image_url, $activity ) {
+		/**
+		 * Whether the organizer image should be imported and attached or not.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool                                        $import_organizer_image Defaults to `true`
+		 * @param int                                         $organizer_id           The organizer post ID
+		 * @param string                                      $image_url              The URL to the image that should be imported
+		 * @param Tribe__Events__Aggregator__Record__Activity $activity               The importer activity so far
+		 */
+		$import_organizer_image = apply_filters( 'tribe_aggregator_import_organizer_image', true, $organizer_id, $image_url, $activity );
+
+		if ( ! $import_organizer_image ) {
+			return false;
+		}
+
 		if ( ! tribe_is_organizer( $organizer_id ) ) {
 			return false;
 		}
 
+		return $this->import_and_attach_image_to( $organizer_id, $image_url, $activity );
+	}
+
+	/**
+	 * Attaches a service-provided image to a venue.
+	 *
+	 * @since TBD
+	 *
+	 * @param int                                         $venue_id The venue post ID.
+	 * @param string                                      $image_url
+	 * @param Tribe__Events__Aggregator__Record__Activity $activity
+	 *
+	 * @return bool Whether the image was attached to the venue or not.
+	 */
+	public function import_venue_image( $venue_id, $image_url, $activity ) {
+		/**
+		 * Whether the venue image should be imported and attached or not.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool                                        $import_venue_image Defaults to `true`
+		 * @param int                                         $venue_id       The venue post ID
```

Comment:
Doesn't matter too much but the spacing/alignment is somewhat out of whack after `$venue_id` and before the description

Reply:
Correct again, was Chaotic Evil, I've corrected to Lawful Good.